WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -137,6 +160,7 @@ private extension CartView {
         static let horizontalPadding: CGFloat = 16
         static let verticalPadding: CGFloat = 8
         static let shoppingBagImageSize: CGFloat = 104
+        static let offSetPositionThreshold: CGFloat = 104
```

Comment:
This one doesn't look robust. 🤔 For example, if we increase or decrease dynamic type - it requires a different threshold. 

---

If we use a local coordinate space for scroll view, offset needs to simply be <0 to mean that we scrolled above header